website/content/docs/configuration/listener/tcp.mdx

```diff
@@ -62,6 +62,10 @@ default value in the `"/sys/config/ui"` [API endpoint](/vault/api-docs/system/co
   [go-sockaddr template](https://pkg.go.dev/github.com/hashicorp/go-sockaddr/template)
   that is resolved at runtime.
 
+- `chroot_namespace` `(string: "admin/")` – Specifies the namespace that will be used
+as the apparent root directory by this listener. When not specified, this will default to
```

Comment:
```suggestion
as the apparent root namespace by this listener. When not specified, this will default to
```

I get the analogy with chroot(2), but I think this may confuse people.

Reply:
Also, although I think it's implicit, it would be good to spell out what happens if the user provides a ns (i.e. we prepend the chroot ns).